### PR TITLE
Update/304 content tweaks

### PR DESCRIPTION
**Pull request by jvntf**
#304

### CODE REVIEW

File: app/assets/scripts/components/indicators/indicator-shipping.js [hunk 1]
```diff
@@ -241,7 +241,7 @@ class ShippingLongForm extends React.Component {
               </Prose>
             </FoldDetails>
             <MediaImage
-              src={`${baseUrl}/assets/graphics/content/la-port.png`}
+              src={`${baseUrl}/assets/graphics/content/indicators/la-ship.png`}
               alt='Wuhan Before and After'
```
Comment (danielfdsilva) [inline app/assets/scripts/components/indicators/indicator-shipping.js:245]: The `alt` text is likely incorrect

**Comment by jvntf** [comment]
@danielfdsilva the requested edit is to change layer names from 'Nitrogen dioxide' -> NO_2

**Comment by danielfdsilva** [comment]
@jvntf For the layer names you can use jsx (`<>No<sub>2</sub></>`) given that they're define client side

**Comment by jvntf** [comment]
@danielfdsilva ok true, i left as a string and parsed before rendering

**Comment by danielfdsilva** [comment]
Not exactly what I had in mind, but it's working properly

**Pull request merged**